src/hooks/useLiveSpecsExt.ts

```diff
@@ -21,30 +21,33 @@ export interface LiveSpecsExtQueryWithSpec extends LiveSpecsExtQuery {
     spec: any;
 }
 
+const defaultResponse: LiveSpecsExtQuery[] = [];
 const queryColumns = ['id', 'writes_to', 'spec_type'];
 const queryColumnsWithSpec = queryColumns.concat(['spec']);
 
-type DraftID = string[] | string | null;
+type EntityID = string[] | string | null;
 
 // TODO (typing) Initially this was all one function but something was messed up
 //    with handling if/if not the spec was returned. So made two different wrapper
 //     functions for the hook
 
 function useLiveSpecsExt(
-    draftId: DraftID,
+    draftId: EntityID,
     specType: ENTITY,
     includeSpec: true
 ): Response<LiveSpecsExtQueryWithSpec>;
 function useLiveSpecsExt(
-    draftId: DraftID,
+    draftId: EntityID,
     specType: ENTITY,
     includeSpec?: false
 ): Response<LiveSpecsExtQuery>;
 function useLiveSpecsExt(
-    draftId: DraftID,
+    draftId: EntityID,
     specType: ENTITY,
     includeSpec?: boolean
 ): Response<LiveSpecsExtQuery> | Response<LiveSpecsExtQueryWithSpec> {
+    console.log('useLiveSpec');
```

Comment:
Are you leaving this `console.log` behind intentionally?

Reply:
nope. removed